model/bindings.go

```diff
@@ -0,0 +1,122 @@
+package model
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	yaml "github.com/cloudfoundry-incubator/candiedyaml"
+	"github.com/docker/libcompose/config"
+	"github.com/docker/libcompose/utils"
+	"github.com/rancher/rancher-compose/preprocess"
+	"github.com/rancher/rancher-compose/rancher"
+	"io/ioutil"
+	"os"
+	"reflect"
+)
+
+type mapLabel map[string]string
+
+//BindingProperty holds bindings
+type BindingProperty struct {
+	Binding
+}
+
+//Binding holds the fields for Binding
+type Binding struct {
+	Scale  int               `json:"scale"`
+	Labels map[string]string `json:"labels"`
+	Ports  []string          `json:"ports"`
+}
+
+//CreateBindingsRancher creates bindings property for rancher-compose.yml
+func CreateBindingsRancher(relativePath string) map[string]BindingProperty {
```

Comment:
name the input parameter should indicate what path it is - maybe something like pathToRancherCompose